cmd/plugin/rpaasv2/cmd/info.go

```diff
@@ -0,0 +1,230 @@
+	data := [][]string{
+		{max, cpuPercentage},
+		{min, memPercentage},
+	}
+	table.AppendBulk(data)
+	table.Render()
+
+	return buffer.String()
+}
+
+func writeAdressesOnTableFormat(adresses []clientTypes.InstanceAddress) string {
+	data := [][]string{}
+	for _, address := range adresses {
+		data = append(data, []string{address.Hostname, address.IP})
+	}
+	var buffer bytes.Buffer
+	table := tablewriter.NewWriter(&buffer)
+	table.SetHeader([]string{"Hostname", "IP"})
+	table.SetRowLine(true)
+	table.SetAutoWrapText(true)
+	table.SetColumnAlignment([]int{tablewriter.ALIGN_LEFT, tablewriter.ALIGN_CENTER, tablewriter.ALIGN_CENTER, tablewriter.ALIGN_LEFT})
+	table.AppendBulk(data)
+	table.Render()
+
+	return buffer.String()
+}
+
+func writeLocationsOnTableFormat(locations []v1alpha1.Location) string {
+	data := [][]string{}
+	for _, location := range locations {
+		data = append(data, []string{location.Path, location.Destination})
+	}
+	var buffer bytes.Buffer
+	table := tablewriter.NewWriter(&buffer)
+	table.SetHeader([]string{"Path", "Destination"})
+	table.SetRowLine(true)
+	table.SetAutoWrapText(true)
+	table.SetColumnAlignment([]int{tablewriter.ALIGN_LEFT, tablewriter.ALIGN_CENTER, tablewriter.ALIGN_CENTER, tablewriter.ALIGN_LEFT})
+	table.AppendBulk(data)
+	table.Render()
+
+	return buffer.String()
+}
+
+func writeTagsOnTableFormat(tags []string) string {
+	data := [][]string{}
+	for _, tag := range tags {
+		data = append(data, []string{tag})
+	}
+	var buffer bytes.Buffer
+	table := tablewriter.NewWriter(&buffer)
+	table.SetRowLine(true)
+	table.SetAutoWrapText(true)
+	table.SetColumnAlignment([]int{tablewriter.ALIGN_LEFT, tablewriter.ALIGN_CENTER, tablewriter.ALIGN_CENTER, tablewriter.ALIGN_LEFT})
+	table.AppendBulk(data)
+	table.Render()
+
+	return buffer.String()
+}
+
+func writeBindsOnTableFormat(binds []v1alpha1.Bind) string {
+	data := [][]string{}
+	for _, bind := range binds {
+		data = append(data, []string{bind.Name, bind.Host})
+	}
+	var buffer bytes.Buffer
+	table := tablewriter.NewWriter(&buffer)
+	table.SetHeader([]string{"App", "Address"})
+	table.SetRowLine(true)
+	table.SetAutoWrapText(true)
+	table.SetColumnAlignment([]int{tablewriter.ALIGN_LEFT, tablewriter.ALIGN_CENTER, tablewriter.ALIGN_CENTER, tablewriter.ALIGN_LEFT})
+	table.AppendBulk(data)
+	table.Render()
+
+	return buffer.String()
+}
+
+func writeInfoOnJSONFormat(w io.Writer, payload *clientTypes.InstanceInfo) error {
+	message, err := json.Marshal(payload)
+	if err != nil {
+		return err
+	}
+
+	fmt.Fprintln(w, string(message))
+	return nil
+}
+
+func runInfo(c *cli.Context) error {
+	client, err := getClient(c)
+	if err != nil {
+		return err
+	}
+
+	info := rpaasclient.InfoArgs{
+		Instance: c.String("instance"),
+		Raw:      c.Bool("raw-output"),
+	}
+
+	infoPayload, _, err := client.Info(c.Context, info)
+	if err != nil {
+		return err
+	}
+
+	if info.Raw {
+		return writeInfoOnJSONFormat(c.App.Writer, infoPayload)
+	}
+
+	tmpl, err := prepareTemplate()
```

Comment:
I'd store this Go template in a "global" variable (unexported); So it will avoid to create the Go template on every run;